### PR TITLE
UXUI-239 new lozenge component

### PR DESCRIPTION
**Pull request by lukehansell-hx**
#### What does this PR do? (please provide any background)
Adds a new lozenge component to the UI-toolkit

#### What tests does this PR have?
I've added new tests for the component in the `src/components/lozenge/__tests__` directory
run `npm test`

#### How can be this tested?
run `grunt docs` and have a play

#### What gif best describes how you feel about this work?

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [ ] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [ ] I’ve checked for coding anti-patterns.
- [ ] I’ve checked for appropriate test coverage.
- [ ] I’ve run all the tests locally and they pass.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

### CODE REVIEW

**Comment by LouisPryer** [comment]
Taking Dev

File: docs/src/Components.jsx [hunk 1]
```diff
@@ -90,6 +90,22 @@ var Components = React.createClass({
...
 
+        <article id="lozenge">
+          <h3>Lozenge</h3>
+          <p>A small pill shaped item for displaying information</p>
+          <CustomComponent codeText={fs.readFileSync(__dirname + '/../examples/Lozenge.jsx', 'utf8')} />
+          <h4>Attributes</h4>
+          <ul>
+            <li><code>tip</code> String - Tooltip text to display</li>
+            <li><code>purpose</code> String - Specifies purpose and adds relevant style</li>
+            <li><code>info</code> Boolean - Adds info style</li>
```
Comment (jodiedoubleday) [inline docs/src/Components.jsx]: In all other components we pass through `purpose='danger'` etc maybe we should keep this consistent

**Comment by LouisPryer** [comment]
:+1: Once comments have been addressed

**Comment by jodiedoubleday** [comment]
Looks good :+1: will need `size` prop adding at a later date.

**Pull request merged**